static/js/jira-select-github-version.js

```diff
@@ -0,0 +1,15 @@
+$('.jiraSelectGitHubVersion__options__card').click(function (event) {
+	event.preventDefault();
+	const selectedCard = event.target.className;
+	const selectedVersion = selectedCard.includes("cloud") ? "cloud" : "server";
+	const className = `.jiraSelectGitHubVersion__options__card.${selectedVersion}`;
+
+	if (selectedVersion === "cloud" && $(".jiraSelectGitHubVersion__options__card.server").hasClass("selected")) {
+		$(".jiraSelectGitHubVersion__options__card.server").removeClass("selected");
+	} else if (selectedVersion === "server" && $(".jiraSelectGitHubVersion__options__card.cloud").hasClass("selected")) {
+		$(".jiraSelectGitHubVersion__options__card.cloud").removeClass("selected");
+	}
```

Comment:
Wouldn't it be easier to just remove 'selected' from all cards, then add it to the appropriate one?

```
$(".jiraSelectGitHubVersion__options__card").removeClass("selected");
$(`${className}`).addClass("selected");
```
No conditionals needed.

Reply:
Nice. Much simpler 😄